src/cascadia/TerminalSettingsEditor/LaunchViewModel.h

```diff
@@ -22,16 +30,28 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
         void CurrentDefaultTerminal(const IInspectable& value);
         winrt::Windows::Foundation::Collections::IObservableVector<Model::DefaultTerminal> DefaultTerminals() const;
 
+        // We cannot use the macro for LaunchMode because we want to insert an event into the setter

```

Comment:
But if you use the `PropertyChanged` at the top of `LaunchViewModel.cpp`, you can right?